integration-tests/maven/src/test/java/io/quarkus/maven/it/ListExtensionsIT.java

```diff
@@ -0,0 +1,74 @@
+package io.quarkus.maven.it;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.*;
```

Comment:
We don't use star imports. Make sure to use the Quarkus code format rules in https://github.com/quarkusio/quarkus/tree/master/independent-projects/ide-config/src/main/resources

Reply:
Done.